Dockerfile

```diff
@@ -71,11 +66,19 @@ RUN yarn install --immutable --check-cache
 # This contains built environments of Calypso. It will
 # change any time any of the Calypso source-code changes.
 ENV NODE_ENV production
-RUN yarn run build
+RUN yarn run build 2>&1 | tee build_log.txt && \
```

Comment:
I found it easier to do this in the Dockerfile. Doing it in teamcity meant moving away from the built-in docker command type, which has some default settings that seem to make it work better.